src/aat/resources/features/F-1008 - Submit Event Creation Handle Case Links/CaseLinkVerifications/F-1008_VerifyCaseLinksBase.td.json

```diff
@@ -0,0 +1,41 @@
+{
+	"title": "should retrieve case links for a specified case reference",
+	"_guid_": "F-1008_VerifyCaseLinksBase",
+
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve case links for an existing case reference",
+
+	"method": "GET",
+	"uri": "/testing-support/case-link/{cid}",
+
```

Comment:
Can we have a **common TD file** to cover the basics of this new API signature 

and see also the following in hmcts/ccd-data-store-api#1738:

* F-1008-VerifyCaseLinks
* F-1009-VerifyCaseLinks
* F-1009-VerifyCaseLinksNotInserted
* F-1009-VerifyCaseLinks_V2
* F-1009-VerifyCitizenCaseLinks
* F-1009-VerifyMultipleCaseLinks
* F-1009-VerifyMultipleCitizenCaseLinks

Reply:
Done in [2e98c6c](https://github.com/hmcts/ccd-data-store-api/pull/1719/commits/2e98c6c69d70f77077a6395aad95e21a9135f99a)